10_installation/10_install_server/12_install_root_server_sles.txt

```diff
@@ -19,11 +19,9 @@ Also, Rudder server requires the +git+ software, that can be found on SLES SDK D
 
 ====
 
-SLES 11 will try to install PostgreSQL 8.3 by default, which is not supported by Rudder and will cause various glitches
-in the web interface, as well as reporting failures.
+SLES 11 pre SP4 will try to install PostgreSQL 8.x by default, which is not supported by Rudder and will cause various glitches in the web interface, as well as reporting failures.
 
-It is really mandatory to either add the OpenSuSE build service repository, or install postgresql91-server (if available)
-beforehand to prevent the system from choosing the default PostgreSQL implementation.
```

Comment:
Shouldn't we change the version in the dependencies to reflect that in the package? It would avoid installing broken versions.